### ISSUE TITLE
Debug: ..watch variable menu option missing

### ISSUE THREAD

**Issue by joneal** [Microsoft/vscode#10614]
- VSCode Version: Code 1.4.0 (6276dcb0ae497766056b4c09ea75be1d76a8b679, 2016-08-04T16:45:31.680Z)
- OS Version: Windows_NT ia32 6.3.9600

Steps to Reproduce:

1.  Debug code (Node.js).
2.  Set breakpoint.
3. When breakpoint is hit, select and right click on variable.
4. Menu options Debug: Evaluate and Debug: Run to Cursor are present, but Debug: Watch variable is now missing.

**Comment by alexandrudima** [comment]
fyi @weinand

**Comment by jaygeek** [comment]
**Yes, someone else is talking about this!**
I was getting tired of waiting for this to _return,_ as it slows down our workflow by having to use the Copy a source variable, Debug window... Add Expression, Paste... Refocus editor on source code, etc.
I got used to: Right-Click, Add Watch like Visual Studio.
**What is the status?** (because it's not even in the Insider's Build yet.)

**Comment by jaygeek** [comment]
A related issue I've noticed is... Oh wait, here it is: https://github.com/Microsoft/vscode/issues/2399

1. Debug code (Node.js).
2. Set breakpoint.
3. When breakpoint is hit, Use the `DEBUG CONSOLE` to assign a new value to the variable. (The value output after assignment _seems_ to indicate it has been changed.)
4. RESULT: Hovering and Watch list do not reflect new value, and...
5. Re-Evaluating the variable now in the `DEBUG CONSOLE` shows the old value  (I swear this worked before in @Code)

It should work **like** Mikaël Mayer's answer here: http://stackoverflow.com/questions/4603021/is-it-possible-to-change-javascript-variable-values-while-debugging-in-google-ch/24694804#24694804

Screenshot in Chrome Tools: http://i.stack.imgur.com/ubAJ2.png
